### PR TITLE
small change

### PR DESCRIPTION
**Pull request by tangyl92**
small change

### CODE REVIEW

File: src/dscigametrics/metric4.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+def find_best_and_worst_campaigns(df, start_date, end_date, campaign_ids, metric):
```
Comment (WeilinHan8) [inline src/dscigametrics/metric4.py]: Could you change your file name to function name? Appreciate it.

**Review by WeilinHan8 — CHANGES_REQUESTED** [review]

File: src/dscigametrics/metric4.py (hunk 1 above)
Comment (WeilinHan8) [reply src/dscigametrics/metric4.py]: All others looks good to me now.

Comment (tangyl92) [reply src/dscigametrics/metric4.py]: happy now?

**Review by WeilinHan8 — APPROVED** [review]
Thanks! This looks nice!

**Pull request merged**